src/test/java/pl/sda/awesomemovies/client/controllers/MovieClientControllerTest.java

```diff
@@ -0,0 +1,87 @@
+package pl.sda.awesomemovies.client.controllers;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.ui.Model;
+import pl.sda.awesomemovies.client.movie.Movie;
+import pl.sda.awesomemovies.client.movie.MovieClientController;
+import pl.sda.awesomemovies.client.movie.MovieClientService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+public class MovieClientControllerTest {
```

Comment:
Na razie chyba wywaliłbym ten test, bo nie weryfikuje nic istotne z perspektywy kodu. W przyszłości, jak dorobimy trochę logiki po stronie serwisu, może warto będzie to otestować.